raiden/transfer/channel.py

```diff
@@ -57,22 +57,23 @@
 from raiden.utils import publickey_to_address, typing
 
 
-def is_known(end_state, hashlock):
-    """True if the `hashlock` corresponds to a known lock."""
+def is_lock_pending(end_state, hashlock):
+    """True if the `hashlock` corresponds to a lock that is pending withdraw
+    and didn't expire."""
```

Comment:
Shouldn't the closing docstring be in its own line here?

Reply:
Good question, yes? My oneline is not really a oneline :)

Reply:
Yeah I guess you wanted this to be an oneliner title but it's not ;)